index.html

```diff
@@ -3053,15 +3053,105 @@ <h2>Identifier</h2>
 
     <section id="sec-http-webhook-profile-message-format">
       <h3>Message Format</h3>
-      <p class="rfc2119-assertion" id="http-webhook-profile-message-format-1">
-        Event notification messages MUST comply with the following data schema.
+      <p>
+        This section defines a JSON object format for events.
       </p>
-
-      <p class="ednote">
-        <!-- TODO -->
-        TODO: Describe data and dataResponse schemas.
+    <p class="rfc2119-assertion" id="http-webhook-profile-message-format-1">
+        Event notification messages MUST comply with the following data schema.
       </p>
 
+        <p>
+          The format is aligned with the <a href="https://cloudevents.io/">cloud events specification</a> 
+          to ensure interoperability with cloud systems that adopt that specification.
+        </p>
+        <p class="note">
+          This event format is currently only used for event affordances, however it could be used for 
+          observable properties as well.
+        </p>
+        <p>
+          An event message contains a set of metadata properties and an optional data payload.
+          The metadata fields allow to determine the type and source of the event as well as
+          a timestamp indicating when the event occured and an optional data payload.
+        </p>
+        <p>
+          <span class="rfc2119-assertion" id="profile-json-event-payload">
+            The following constraints MUST be
+            applied to the event payload:
+          </span>
+        </p>
+        <table class="def">
+          <thead>
+            <tr>
+              <th>keyword</th>
+              <th>type</th>
+              <th>type constraints</th>
+              <th>value constraints</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>specversion</td>
+              <td>string</td>
+              <td>mandatory, non-empty</td>
+              <td>The <code>specversion</code> MUST have the value "1.0"</td>
+            </tr>
+            <tr>
+              <td>id</td>
+              <td>string</td>
```

Comment:
TD uses anyURI

Reply:
good point - we could restrict the string contents to URI formats only.